packages/xo-server/src/api/backup.js

```diff
@@ -62,7 +65,7 @@ function handleFetchFiles (
 
       const archive = archiver(archiveFormat)
       archive.on('error', error => {
-        console.error(error)
+        log.error({ error })
```

Comment:
I think we should support two types of calls:
- `log.*(message: string, data: object)`
- `log.*(value: unknown)` and I propose to handle the case of `Error` and similar objects for now.

Reply:
I just changed the log function. Normally, it's good. If a message is specified, it's displayed with the error after. Otherwise, the error is displayed with `"an error as occured"` if `error.message` is not defined.